setup.py

```diff
@@ -42,8 +43,9 @@
         'rmq': [
             'pika>=1.0.0b2',
             'topika>=0.2.0, <0.3.0',
-            'tornado>=4; python_version<"3"',
-            'tornado<5; python_version>="3"',
+            # 'tornado>=4; python_version<"3"',
```

Comment:
Could you delete these commented out lines?  The tornado range you give seems fine.

Reply:
I'll remove them.